ReSharper.FSharp/src/FSharp.Psi.Features/src/Parsing/FSharpTreeBuilderBase.fs

```diff
@@ -74,7 +74,7 @@ type FSharpTreeBuilderBase(lexer, document: IDocument, lifetime, path: VirtualFi
         x.Mark()
 
     member x.Mark() =
-        /// The base member is protected and cannot be used in closures.
+        // The base member is protected and cannot be used in closures.
```

Comment:
I'm sorry, but could you please not fix this warning in this PR? 🙂
Let's think about having a quick fix for it instead, having this as a reminder.